string id
string type
string time
string emergency_cmd
